addons/web/static/src/search/search_model.js

```diff
@@ -1393,6 +1397,7 @@ export class SearchModel extends EventBus {
         return {
             controlPanel: "controlPanel" in display ? display.controlPanel : {},
             searchPanel:
+                this.sections.size &&
```

Comment:
seems that it should be part of another commit

Reply:
shouldn't this be tested?